pkg/scheduler/plugins/elastic/elastic.go

```diff
@@ -29,22 +29,22 @@ func JobOrderFn(l, r interface{}) int {
 	lv := l.(*podgroup_info.PodGroupInfo)
 	rv := r.(*podgroup_info.PodGroupInfo)
 
-	lvAllocatedCount := int32(lv.GetActiveAllocatedTasksCount())
-	rvAllocatedCount := int32(rv.GetActiveAllocatedTasksCount())
+	lvAtLeastOneSubgroupUnderMinAvailable, lvAllSubgroupsAboveMinAvailable, lvAllSubgroupsExactlyMinAvailable := minAvailableState(lv)
```

Comment:
What do you think about making this part more general by reducing subgroup related terminology.
For example:
```lvBelowMinAvailable, lvAboveMinAvailable, lvExactlyAtMinAvailable := minAvailableState(lv)```
I think that subgroups is an implementation details in the scope of this part of the code